fdk.go

```diff
@@ -129,10 +142,63 @@ type HTTPContext interface {
 	RequestMethod() string
 }
 
+// TracingContext contains all configuration for a function invocated to
+// get the tracing context data.
+type TracingContext interface {
+	/**
+	 * Returns true if tracing is enabled for this function invocation
+	 * @return whether tracing is enabled
+	 */
+	IsTracingEnabled() bool
```

Comment:
This is just called isEnabled in the NodeFDK. Dont know about the others. Just a nit.

Reply:
In the python FDK it's is_tracing_enabled

Reply:
Even in java it is isTracingEnabled

Reply:
Node looses then! I guess that slipped through. Hard to get the different language interfaces exactly the same. The term trace seemed a bit redundant in the node context: so it is 'traceContext.isEnabled' not 'traceContext.isTraceEnabled'.

Reply:
Confused should I do this change or not